stan/math/prim/fun/quad_form.hpp

```diff
@@ -13,21 +13,21 @@ namespace math {
  * Symmetry of the resulting matrix is not guaranteed due to numerical
  * precision.
  *
- * @tparam RA number of rows in the square matrix, can be Eigen::Dynamic
- * @tparam CA number of columns in the square matrix, can be Eigen::Dynamic
- * @tparam RB number of rows in the second matrix, can be Eigen::Dynamic
- * @tparam CB number of columns in the second matrix, can be Eigen::Dynamic
- * @tparam T type of elements
+ * @tparam EigMat1 type of the first (square) matrix
+ * @tparam EigMat2 type of the second matrix
  *
  * @param A square matrix
  * @param B second matrix
  * @return The quadratic form, which is a symmetric matrix of size CB.
```

Comment:
Doc needs to be updated, since there isn't a ```CB``` tparam any more

Reply:
fixed